build/build.go

```diff
@@ -307,8 +307,10 @@ func parseAndAugment(pkg *build.Package, isTest bool, fileSet *token.FileSet) ([
 		for _, decl := range file.Decls {
 			switch d := decl.(type) {
 			case *ast.FuncDecl:
+				// Allow overridden function calls
+				// The standard library implementation of foo() becomes _gopherjs_overridden_foo()
 				if replacedDeclNames[funcName(d)] {
-					d.Name = ast.NewIdent("_")
+					d.Name.Name = "_gopherjs_overridden_" + d.Name.Name
 				}
 			case *ast.GenDecl:
 				switch d.Tok {
```

Comment:
That's right, here is the updated comment: https://github.com/gopherjs/gopherjs/pull/558/commits/febab31b01360451d242cd77388b1aaac0f2c828